client/app/bundles/course/survey/pages/ResponseShow/index.jsx

```diff
@@ -110,7 +110,7 @@ class ResponseShow extends React.Component {
 
   renderUnsubmitButton() {
     const { response, flags: { canUnsubmit, isLoading } } = this.props;
-    if (!canUnsubmit || isLoading) { return null; }
+    if (!canUnsubmit || isLoading || !response.submitted_at) { return null; }
```

Comment:
As discussed, this is a matter of style. I put here because it is easier to understand - e.g. when I'm thinking about the unsubmit button, I can find the render logic relating to it in a single place.